source/fundamentals/stable-api.txt

```diff
@@ -51,10 +56,10 @@ version of the {+stable-api+}.
 
 .. tip::
 
-   If you need to run commands using more than one version of the
-   {+stable-api+}, instantiate a separate client with that version.
+   To run commands using more than one version of the {+stable-api+},
+   instantiate a separate client with that version.
```

Comment:
S: clarity

```suggestion
   You must create a new client for each version of the {+stable-api+} on
   which you need to run a command.
```

(I think this is accurate? ^^^^)

Reply:
seems right to me, changed!